### DISCUSSION TITLE
Régió lista

### DISCUSSION THREAD

**Discussion by hollosipeti** [amaximus/met_alerts_hu#10] [General]
Szia!

Előszöris gratula és köszi! Két integrációdat is használom.
Az lenne a kérdésem, hogy régió listát hol lehet elérni? Mert szeretném átállítani.

Köszi!

**Comment by amaximus** [comment]
Szia,  ha a region_id-ra gondolsz akkor azt az alábbi linken levő térképen https://www.met.hu/idojaras/veszelyjelzes/index.php a régióra rávíve az egérkurzort (nem kell rákattintani) a linkben szereplő kód.

**Reply by hollosipeti** [reply]
Megvan! Köszi!
